### ISSUE TITLE
Can't set another language for menus

### ISSUE THREAD

**Issue by GHemis** [qTranslate-Team/qtranslate-x#374]
Hi there,
Until yesterday, I had a set of menu that was in French and English, using the qtranslate-x plugin. 
I updated some of the names of my menu in English and when I applied modifications, all the menus, in both languages, turned to be in English. I tried to correct my menus in French, and when I applied modifications, both set of menus were in French.

I updated my Wordpress version from 4.4.2 to 4.5 yesterday and I suspect this must be the reason why.

- Julie

**Comment by richpav** [comment]
The problem will appear only if you change the label of any menu item after upgrading to 4.5. Then all the labels, not just the one(s) you changed, turn monolingual.

The workaround is to disable qtranslate-x, change each of the menu labels strings to what they should be, for example [:en]Service[:ja]サービス[:], then re-enable the plugin.

**Comment by acenkus** [comment]
Yes! In our prog. team whe have the same issue!

**Comment by wocmultimedia** [comment]
Disabling and reactivating qtranslate-x will reanable the correct menu or do I have to reset again the double language for each item. 
- I got in my test site which has debug on this warning and maybe is related to your plugin
add_utility_page is deprecated since version 4.5! Use add_menu_page() instead. in /web/htdocs/www.wocmultimedia.biz/home/test_site/wp-includes/functions.php on line 3662

**Comment by BigJonMX** [comment]
![bug - qtranslatex1](https://cloud.githubusercontent.com/assets/18481586/14558449/c75db5c4-0302-11e6-9eea-7c0624cf5fa2.jpg)

I discovered its not necessary to disable the plugin. Just edit all the labels and press save once.
Note: ALL labels have to be done again at the same time - even if only one needs chaning.

**Comment by wocmultimedia** [comment]
Do you mean you have to write the quick tags too? Because I changed the label and after saving they show same word for both language

**Comment by wocmultimedia** [comment]
I found a good workaround. The way I found is, go to Settings > Languages > Advanced and choose RAW instead of the LSB. Then go to Appearance > Menus and rewrite the Item labels. You can copy the title with quicktags from Original and paste in the Navigation Label then Save.

**Comment by GHemis** [comment]
Thank you @wocmultimedia, using the RAW option is definitely the fastest way to deal with it!

**Comment by acenkus** [comment]
I do not understand, why this ticket is closed? Because it works without LSB is not a bug?!!! It still a **problem**.

**Comment by jmeile** [comment]
Hi,

It is actually open. See the status after the issue's title on the top of this page. That "Closed" you are seeing refers to the issue #377, which is a duplicate of this one.

I'm also affected by this bug and waiting for a solution. I think the best is that you subscribe to the issue.

Best regards
Josef

**Comment by davidrio7** [comment]
> I found a good workaround. The way I found is, go to Settings > Languages > Advanced and choose RAW instead of the LSB. Then go to Appearance > Menus and rewrite the Item labels. You can copy the title with quicktags from Original and paste in the Navigation Label then Save.
After adjusting the Menu Items can revert to LSB for an easier post compilation

> 
thank you wocmultimedia for this. Since I am a newbie could you please explain how do I copy the title with quicktags from original? 
Thanks

**Comment by wocmultimedia** [comment]
@davidrio7 Hi, it's easy. You go to the menu, click on the little triangle on the topright of the item you want to change and it opens. In this panel you see the field named "Original" go with the mouse to the end of the line where you find the [:] click and move to the left to select and copy like any other text.
Then you go to Navigation Label and paste it. That's all.

**Comment by BigJonMX** [comment]
I find it easier to just type it in...
[:en]Welcome[:de]Willkommen[:es]Bienvenida[:]

you see the pattern 
[:country-code] words [:another-country-code] words [:]
and just a colon to finish

**Comment by davidrio7** [comment]
do you think Polylang is easier to work with? I just installed it on another website I  run and it seems ok

**Comment by wocmultimedia** [comment]
Are you looking for a solution with qtranslate-x or just playing around.
If you need to use qtranslate-x just make the use of one of the workarounds till the bug is fixed and go on working, Sparing one or two minutes is nothing respect of having the site messed.
My workaround was ok for me cause I have more than 50 items and titles were longer and different from what appeared on navigation label, so copying from original helped me to save time while to other people not. Don't worry and before creating a new site make several tests with plugins none is perfect.

**Comment by johnclause** [comment]
There is an updated information in https://wordpress.org/support/topic/issues-with-wordpress-45

**Comment by johnclause** [comment]
Please, if you guys have some time, help us to test the fix asap, as described in [WP thread](https://wordpress.org/support/topic/issues-with-wordpress-45).

**Comment by BigJonMX** [comment]
This is getting rather annoying now!
I had to update one link in my menu, and oops i saved it. So now i have to spend an hour re-translating and typing in the whole damn thing...
Is this what i can expect from wordpress? (i've only recently moved over from coding my own sites).

So next time WP says there's an update, i should wait 6 months for the bugs to be ironed out? 
Its like we used to only update windows once the service pack came out.
